src/index.ts

```diff
@@ -104,9 +104,15 @@ class Variate {
         };
 
         // Targeting information
-        const targeting = get(value, 'targeting', {default: {}});
+        const customTargeting = get(value, 'targeting', {default: {}});
 
-        this._env = Object.assign({}, this._env, {view, viewport, targeting});
+        this._env = Object.assign({}, this._env, {
```

Comment:
Wondering if we can not spread here to assign to a new object. 

```
{
    ...this._env,
    {
            view,
            targeting: {
                ...targeting,
                ...customTargeting
            }
    }
}

Reply:
We could, but I like the Object.assign to have control on the direction of merging.